tests/API_RH.py

```diff
@@ -34,12 +34,16 @@ def extract_jobs_title(soup):
 
 
 def create_soup(req):
-    print(req.status_code)
     soup = BeautifulSoup(req.content, 'html.parser')
     # print(soup.prettify())
     return soup
 
 
+def check_status_code(req):
+    print(req.status_code)
```

Comment:
What is the purpose of this function as opposed to just using req.status_code directly?

Reply:
same answer as above